src/main/java/core/basesyntax/serviceimpl/CsvWriterImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.serviceimpl;
+
+import core.basesyntax.service.Writer;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.Collections;
+
+public class CsvWriterImpl implements Writer {
+    @Override
+    public void write(String content, String filePath) {
+        File output = new File(filePath);
+        try {
+            output.createNewFile();
```

Comment:
you can use FIles-class without File-class, and it will create your output file automatically